### PR TITLE
Add Sublocality level

### PR DESCRIPTION
**Pull request by TNAJanssen**

### CODE REVIEW

File: src/Geocoder/Model/SubLocalityLevel.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Geocoder\Model;
+
+/**
+ * @author William Durand <william.durand1@gmail.com>
```
Comment (Nyholm) [inline src/Geocoder/Model/SubLocalityLevel.php]: Not sure we should give Willian credit here. Write your name =)

File: src/Geocoder/Model/Address.php [hunk 2]
```diff
@@ -56,6 +56,11 @@
     private $adminLevels;
 
     /**
+     * @var AdminLevelCollection
```
Comment (Nyholm) [inline src/Geocoder/Model/Address.php]: Wrong annotation

File: src/Geocoder/Model/Address.php [hunk 3]
```diff
@@ -66,34 +71,43 @@
...
+     * @param string                                     $postalCode
+     * @param string                                     $locality
+     * @param string                                     $subLocality
+     * @param \Geocoder\Model\AdminLevelCollection       $adminLevels
+     * @param \Geocoder\Model\SubLocalityLevelCollection $subLocalityLevels
+     * @param \Geocoder\Model\Country                    $country
+     * @param null                                       $timezone
      */
     public function __construct(
-        Coordinates $coordinates          = null,
```
Comment (Nyholm) [inline src/Geocoder/Model/Address.php]: Why would you change the formatting here?

**Comment by TNAJanssen** [comment]
@Nyholm changed your comments and branch on pull https://github.com/geocoder-php/Geocoder/pull/585

**Pull request closed without merging**